pkg/controller/azure/storage/container/container.go

```diff
@@ -0,0 +1,309 @@
+/*
+Copyright 2018 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"context"
+	"log"
+	"reflect"
+	"time"
+
+	"github.com/Azure/go-autorest/autorest/to"
+
```

Comment:
any reason for the blank line between most of the imports in this file?

Reply:
Fixed.